docs/intro/index.md

```diff
@@ -14,11 +14,11 @@
     <img :src="$withBase('/logoBlack.png')" alt="foo" />
 </div>
 
-Welcome to our own Gorilla Logic Vue Bootcamp. As many of you, the team who created this Bootcamp found interesting learning Vue and/or got involved in this path due to a project need. This is great, but the amount of information and learning paths are not that great.
+Welcome to our own Gorilla Logic Vue.js Bootcamp. As many of you, the team who created this Bootcamp found interesting learning Vue and/or got involved in this path due to a project need. This is great, but the amount of information and learning paths are not that great.
 
 Thousands of gigabytes of information are not enough to define the amount of information we can find on the internet, but many of these places (not even the offical documentation) tells you the right or recommended path to learn Vue.js.
 
-Bootcamp's main target is not only to provide you with a proper path to learn Vuejs but also with a methdology that allows you to feel confortable while learning about the framework through all the different challenges you will find right away.
+Bootcamp's main target is to provide you with a proper path to learn this framework, but also, with a methdology that allows you to feel confortable with the framework after passing through all the different challenges you will find right away.
```

Comment:
Using _framework_ x2 sounds repetitive. I rather prefer using _vuejs_, then _framework_ as in previous change.

_Methodology_  is misspelled

Reply:
done